### PR TITLE
Test literal edge cases

### PR DESCRIPTION
**Pull request by greatestape**
- drop unused and unreachable error case
- add tests for bad edges
- test Point repr to get that last 1% of code coverage

### CODE REVIEW

File: asciigraf/asciigraf.py [hunk 1]
```diff
@@ -15,10 +15,10 @@ def graph_from_ascii(network_string):
     """ Produces a networkx graph, based on an ascii drawing
         of a network
     """
-    EDGE_CHAR_NEIGHBOURS = {
+    EDGE_CHAR_NEIGHBOURS = {  # first point in tuple is the point parsed first
         "-":  [Point(-1, 0), Point(1, 0)],
         "\\": [Point(-1, -1),  Point(1, 1)],
-        "/":  [Point(-1, 1), Point(1, -1)],
```
[the remark this answers is not in the document]
Comment (greatestape) [reply asciigraf/asciigraf.py]: I wanted to order all the points so the first one would always be the one we'd already have parsed. So that it's the "leading" neighbour. It's sort of a fussy detail.

**Comment by greatestape** [comment]
resolves #15
